### DISCUSSION TITLE
automatical adding to list from system cache and "confirm delete"

### DISCUSSION THREAD

**Discussion by bloke90** [TumblThreeApp/TumblThree#304] [Q&A (Question & Answer)]
don't know who was distrurbed but is it possible to allow user to decide by "setting" if we want to automatically copy the cache website to the list (as it was all the versions before)

Also the confirmation on deleting any blog from list should be a setting?

Thanks

**Comment by thomas694** [comment]
I'm not sure if I understood you correctly.
Do you mean the feature to automatically add blog urls from the clipboard to the blog list? Just press on `Check Clipboard` to activate it again (button stays checked).

You can find the option `Display confirmation dialogs?` in the settings.

**Comment by thomas694** [comment]
The issue has been closed. You can still comment. Feel free to ask for reopening the issue if needed.
